shared/desktop/remote/sync-props.desktop.js

```diff
@@ -5,69 +5,117 @@
 import * as React from 'react'
 import * as SafeElectron from '../../util/safe-electron.desktop'
 
+// set this to true to see details of the serialization process
+const debugSerializer = __DEV__ && false
```

Comment:
Could we put this in `local-debug`

Reply:
i doubt anyone will ever use this so its not a general thing you can control externally. we do this in a bunch of places for not-often-used settings